UITests/BaseTest.cs

```diff
@@ -0,0 +1,91 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using FlaUI.UIA3;
+using FlaUI.Core;
+using System;
+using System.Threading.Tasks;
+using EnvDTE;
+using AutomationWindow = FlaUI.Core.AutomationElements.Window;
+
+namespace UITests
+{
+   [TestClass]
+   public class BaseTest
+   {
+       protected static UIA3Automation _automation;
+       protected static Application _app;
+       protected static AutomationWindow _mainWindow;
+       protected static DTE _dte;
+
+       [ClassInitialize]
+       public static void ClassInitialize(TestContext context)
+       {
+           // Initialize automation and launch VS
+           _automation = new UIA3Automation();
+           _app = Application.Launch("devenv.exe");
+
+            // Wait for launch VS
+            Task.Delay(15000).Wait();
+            _mainWindow = WaitForMainWindow();
+
+           // Handle initial setup
+           SetupVisualStudio().Wait();
+       }
+
+       private static async Task SetupVisualStudio()
```

Comment:
method is marked as async but is called with .Wait(), which can cause deadlocks. It's better to use await directly if possible